pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -191,8 +191,8 @@ type priorityQueueOptions struct {
 	podMaxBackoffDuration             time.Duration
 	podMaxInUnschedulablePodsDuration time.Duration
 	podNominator                      framework.PodNominator
-	clusterEventMap                   map[framework.ClusterEvent]sets.String
 	preEnqueuePluginMap               map[string][]framework.PreEnqueuePlugin
+	clusterEventMap                   map[framework.ClusterEvent]sets.Set[string]
```

Comment:
I think there are some checks that have not been approved. Could you check them out?

Reply:
What do you mean checks? I ran go test here and it passed.